src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -244,7 +244,6 @@
 	font-family: var(--monaco-monospace-font);
 	font-size: 12px;
 	color: var(--vscode-textPreformat-foreground);
-	background-color: var(--vscode-textPreformat-background);
```

Comment:
This is required to ensure that `code` blocks are visible in chat after updating `vscode-textPreformat-foreground`